package.json

```diff
@@ -37,6 +39,7 @@
     "find-parent-dir": "^0.3.0",
     "fs": "^0.0.1-security",
     "googleapis": "^25.0.0",
+    "grpc": "^1.10.1",
```

Comment:
Is grpc needed? Where do you use it?

Reply:
Oops, was having a weird error earlier and that fixed it. Seems to be fine without it now.